demos/README.md

```diff
@@ -0,0 +1,7 @@
+# `@nucypher/*` Demos
+
+- [`taco-demo`](./taco-demo) - A demo of the `@nucypher/taco` library.
+- [`taco-nft-demo`](./taco-nft-demo) - A demo an NFT-based condition using the `@nucypher/taco` library.
+- [`nucypher-ts-demo`](https://github.com/nucypher/nucypher-ts-demo) - A demo of PRE in the `nucypher-ts` library.
+- [`tdec-sandbox`](https://github.com/nucypher/tdec-sandbox) - A demo of tDec in the `nucypher-ts` library.
+- [`tdec-nft-example`](https://github.com/nucypher/tdec-nft-example) - A demo of tDec in the `nucypher-ts` library.
```

Comment:
In the spirit of the monorepo, shall we also internalize these demos to this repo?

Reply:
Internalizing `nucypher-ts-demo` will require releasing `@nucypher/pre`, which I think we should postpone. `tdec-sandbox` and `tdec-nft-example` will be deprecated (archived) and replaced by existing `demos`.

Reply:
I also generally agree with the division of examples/demos mentioned above 👍🏻

Reply:
> tdec-sandbox and tdec-nft-example will be deprecated (archived) and replaced by existing demos.

^ If we are going to deprecate/archive these, should we remove these in the readme then?